iocore/net/quic/QUICPacket.h

```diff
@@ -307,7 +320,7 @@ class QUICPacketHeaderDeleter
   }
 };
 
-class QUICPacket
+class QUICPacket : public QUICTrackablePacket
```

Comment:
I'm ok with this for now, but I may change it later like:
QUICReceivingPacket : public QUICPacket
QUICSendingPacket : public QUICPacket, QUICTrackablePaket